handsontable/types/settings.d.ts

```diff
@@ -214,6 +214,7 @@ export interface GridSettings extends Events {
   strict?: boolean;
   tableClassName?: string | string[];
   tabMoves?: CellCoords | SimpleCellCoords | ((event: KeyboardEvent) => CellCoords | SimpleCellCoords);
+  themeName?: string;
```

Comment:
Can I ask you to add tests for that option?

Reply:
This option was introduced in https://github.com/handsontable/handsontable/pull/11144, so I guess the typing tests should also be added there? I probably shouldn't have added the type here either 🙄